### DISCUSSION TITLE
Workspaces feature should be optional

### DISCUSSION THREAD

**Discussion by codygarver** [elementary/gala#232] [Ideas]
Would be nice to be able to disable this feature, it's not great on multi-monitor setups because I treat them as static workspaces.

**Comment by fmasa** [comment]
You can disable dynamic workspaces and set workspace amount to 1
```
gsettings set org.pantheon.desktop.gala.behavior dynamic-workspaces false
gsettings set org.gnome.desktop.wm.preferences num-workspaces 1
```

**Comment by gumbogal** [comment]
I can't understand why you would want to be able to disable workspace functionality. They are totally unobtrusive and easy to avoid should you not want to use them.

**Comment by Blast-City** [comment]
I think that the option to disable the Fullscreen/Maximized Workspaces feature makes this issue obsolete since not only elementary behaves like static workspaces, but is also not needed to install additional software to be able to show the desktop/hide all windows.
